tool/plugin/lib/plugin.dart

```diff
@@ -530,24 +531,46 @@ class BuildCommand extends ProductCommand {
       }
 
       // TODO(devoncarew): Remove this when we no longer support AS 3.1.
-      var processedFile = new File(
-          'flutter-studio/src/io/flutter/module/FlutterDescriptionProvider.java');
-      var oldSource;
+      var processedFile1, processedFile2, oldSource1, oldSource2, newSource;
```

Comment:
If we end up with more of these, we'll likely want to wrap in a class, and have something along the lines off a list of them that we can `appy()` and `revert()`.